### ISSUE TITLE
[BUG] Weird Navbar / Mat-ToolBar behavior after adding core theme

### ISSUE THREAD

**Issue by Trianglium** [Trianglium/CandyStore#10]
## Weird Navbar/Mat-toolbar behavior after updating Angular Material core theme
Basically mat-toolbar is black now and responsiveness is acting weird.

### Example of Issue 
![navbar_behavior_after_updating_core_theme](https://user-images.githubusercontent.com/70343977/182382365-5a2c58cb-87e2-494c-b7a5-ab039512bf06.png)
behavior

### Expected behavior
![sweetshoppe-tablet-menu-text](https://user-images.githubusercontent.com/70343977/182382398-82be1917-a681-40d9-9f1d-257c1ac6d648.png)
